Dockerfile

```diff
@@ -5,23 +5,23 @@ FROM tiangolo/uvicorn-gunicorn-fastapi:python3.8
 RUN sed -i 's/TLSv1.2/TLSv1.0/g' /etc/ssl/openssl.cnf
 
 # Install poetry
-RUN cd /tmp && \
-    curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py > get-poetry.py && \
-    POETRY_HOME=/opt/poetry python get-poetry.py --version 1.0.8 && \
-    cd /usr/local/bin && \
-    ln -s /opt/poetry/bin/poetry && \
-    poetry config virtualenvs.create false
+RUN pip install "poetry==1.0.8"
```

Comment:
Poetry recommends not using pip, see:
https://python-poetry.org/docs/: "Poetry provides a custom installer that will install poetry isolated from the rest of your system by vendorizing its dependencies. This is the recommended way of installing poetry."

Reply:
Okay I will roll back the change

Reply:
It says it adds some of it's own dependancies with pip install which may cause conflicts.